lib/tld.js

```diff
@@ -141,7 +141,7 @@ tld.prototype.getRulesForTld = function getRulesForTld (tld, default_rule) {
   }
 
   // Nothing found, apply some default value
-  if (!rules) {
+  if (rules === void 0) {
```

Comment:
OK gotcha :-)

This project code is enforced in strict mode so `undefined` will be rewritten in its scope. In addition, if `rules === false`, we do consider we have no rules.

A more accurate condition in that case would be `if (Array.isArray(rules))` as ES5 is our minimal requirement.